openwisp_monitoring/monitoring/templates/admin/confirmation.html

```diff
@@ -0,0 +1,51 @@
+0,0 +1,50 @@
+{% extends 'admin/base_site.html' %}
+{% load i18n l10n admin_urls static %}
+
+{% block extrahead %}
+    {{ block.super }}
+    {{ media }}
+    <script src='{% static 'admin/js/cancel.js' %}'></script>
+{% endblock %}
+
+{% block bodyclass %}{{ block.super }} app-{{ opts.app_label }} model-{{ opts.model_name }} delete-confirmation
+  delete-selected-confirmation{% endblock %}
```

Comment:
Please put {% endblock %} in a single line as usual

Reply:
Honestly speaking, I had copy-pasted it from the actual Django file and made very few changes. So that's why there's some differences with the style format :sweat_smile: